### PR TITLE
add Op Squeeze on Node.js

### PR DESCRIPTION
**Pull request by lionkunonly**
Add the Op Squeeze on Node.js. Related test cases are passed.

image

### CODE REVIEW

File: node/src/ops/Squeeze.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ops/Squeeze.h"
+
+#include <iostream>
```
Comment (fujunwei) [inline node/src/ops/Squeeze.cpp:17]: Remove it.

**Review by fujunwei — APPROVED** [review]
LGTM with a comment.

File: node/src/ops/Squeeze.cpp (hunk 1 above)
Comment (lionkunonly) [reply node/src/ops/Squeeze.cpp:17]: Fixed

**Comment by fujunwei** [comment]
Merge it, thanks much.

**Pull request merged**